lesson13/main.js

```diff
@@ -65,8 +70,21 @@ const init = async () => {
   createNewList(val);
 };
 
-const button = document.getElementById("js-button");
-button.addEventListener("click", () => {
+openModal.addEventListener("click", () => {
+  modal.classList.remove("hidden");
+  mask.classList.remove("hidden");
+});
+
+closeModal.addEventListener("click", () => {
+  modal.classList.add("hidden");
+  mask.classList.add("hidden");
+});
+
+requestBtn.addEventListener("click", () => {
   init();
-  button.style.display = "none";
+  requestBtn.style.display = "none";
+});
+
+mask.addEventListener("click", () => {
```

Comment:
👍

Reply:
Thank you so much!
And I misunderstood what this lesson was about. I reimplemented in this lesson below.

1. Implemented a modal having a button
2. After clicking this button,  it start fetching the data and hiding this modal
3. Fetched data are displayed on the screen

Reply:
you are correct.

Reply:
you can reference blow site.
https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716

style used to cording style

from 
```js
function a (){}
```
to
```js
function a(){}
```
etc,

Reply:
@kenmori
Thank you for teaching. I'm going to check this link and learn.
I'm appriciated.